packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -20,10 +24,20 @@ export function TelemetryView(): React.ReactElement {
 	const [telemetryEvents, setTelemetryEvents] = React.useState<ITelemetryBaseEvent[]>([]);
 
 	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages related to telemetry.
+		 */
+
+		// incoming telemetry event handler
```

Comment:
We probably want to remove this comment and the blank line preceeding it, since we have a TSDoc comment above already.